src/Tribe/Admin/Organizer_Chooser_Meta_Box.php

```diff
@@ -50,7 +54,18 @@ protected function get_event( $event = null ) {
 	public function render() {
 		$this->render_dropdowns();
 		$this->render_add_organizer_button();
-		include $this->tribe->pluginPath . 'src/admin-views/new-organizer-meta-section.php';
+
+		if ( class_exists( 'Tribe__Events__Community__Main' ) ) {
```

Comment:
Wondering: should core need to be aware of Community's existence in this way and could the same result be achieved by making the template path filterable (which Community could then take advantage of)?